### ISSUE TITLE
PSMOVEService with Portable config (no %appdata in Windows)

### ISSUE THREAD

**Issue by rebitti** [cboulay/PSMoveService#244]
Hello, its possible to run Psmoveservice with a custom directory to put all configurations instead %appdata ?

Example: using a option like -p (portable) and indicate a specific folder ?

Will be nice to have the folder structure documented in the wiki too, with how psmoveservice works with your files.

at,
rebit

**Comment by HipsterSloth** [comment]
Yeah that's pretty reasonable. The current config path is hard coded here:

https://github.com/cboulay/PSMoveService/blob/master/src/psmoveservice/PSMoveConfig/PSMoveConfig.cpp#L29-L53

Totally agreed I should document what the config file structure is in the wiki.
